src/test-utils/testing-library.tsx

```diff
@@ -1,26 +1,19 @@
 import React from 'react';
-import { render as rtlRender } from '@testing-library/react';
-import { IntlProvider } from 'react-intl';
+import { render } from '@testing-library/react';
```

Comment:
you didnt run into any issues trying to import `render` and also trying to cast the name of `renderConnected` as `render` also?

Reply:
nope works good 👍

Reply:
this is the file we use in dev console. I was originally going to add `textComponent="span"` prop to the `IntlProvider` in this file because there was a breaking change in the react-intl upgrade that removed wrapping the text with a `span` but since we didn't add it in the initial upgrade and it hasn't broken anything yet, doesn't seem like it's necessary now